scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -69,7 +69,7 @@ const loadSubItemTitles = (manifest) => {
 
```

Comment:
I'm a little wary of magic strings, but I think there is a case for that here. Can you at least define it as a variable at top and reference that? @jonhartzler is this going to affect indexing?
```
const stringSplit = '::'
```

Reply:
yeah.   I think that this is fine for now.  I think that we really are going to still have to work on this section in the future.  I suspect the real problem is that we will probably need to not do this simple blob field index and if we do we may have to do some other system for the highlighting or I guess we could format the blob with HTML and display it.  anyway  I think it moves this forward.